File: app/src/main/java/me/varlez/movies/common/model/SearchResults.java
package me.varlez.movies.common.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * POJO holding the search results.
 */
public class SearchResults {

    @SerializedName("Search")
    List<Movie> movies;

    public SearchResults(List<Movie> movies) {
        this.movies = movies;
    }

    public List<Movie> getMovies() {
        return movies;
    }

}

File: app/src/main/java/me/varlez/movies/common/model/SearchResults.kt
package me.varlez.movies.common.model

import com.google.gson.annotations.SerializedName

/**
 * POJO holding the search results.
 */
class SearchResults(movies: List<Movie>) {

    @SerializedName("Search")
    var movies: List<Movie>
        internal set

    init {
        this.movies = movies
    }

}
